.changeset/quiet-rice-brake.md

```diff
@@ -2,7 +2,8 @@
 '@shopify/theme-language-server-common': minor
 ---
 
-Support HTML attribute auto-completion
+Improve HTML attribute auto-completion
 
 - Add `=""` to attributes that have values
-- Override whole attribute even if cursor is in the middle of it
+  - Ignore if it attribute also has value before auto-completion is triggered
```

Comment:
```suggestion
```

I don't think we need to document this in the changelog. Feels more like "this is how it's supposed to work" thing than it is a feature (I think?) 

Maybe reword to `Snippet complete HTML attributes (Add `=""`) when relevant`?